cai_causal_graph/time_series_causal_graph.py

```diff
@@ -396,7 +396,7 @@ def get_summary_graph(self) -> CausalGraph:
         Return a summary graph.
 
         Collapse graph in time into a single node per variable (column name).
-        This can become cyclic and bi-directed as X-1 -> Y and Y-1 -> X would become X <-> Y.
+        This can become cyclic and bi-directed as `X-1 -> Y` and `Y-1 -> X` would become `X <-> Y`.
```

Comment:
In summary we do `X(t-1) -> Y` but here we do `X-1 -> Y`. Should we make consistent?

Reply:
Yes, I prefer `X(t-1)`, will change now